docs/enhancements/tenancy-based-auth.md

```diff
@@ -0,0 +1,83 @@
+# Tenancy Based Authorization
+
+## Summary
+The current authorization philosophy grants access to the resource's owner. The user should have a BareMetalCluster create access right in ACM.
```

Comment:
why ACM? Not AMS?

Reply:
Should be AMS. And in truth I think OCM is the right acronym that is used through documentation.

Reply:
AMS is part of OCM

Reply:
Fixed for the entire document. It was a writer error